### PR TITLE
Centerbox setting titles and descriptions

### PR DESCRIPTION
**Pull request by scottcwilson**
Clearing up some ambiguous titles and descriptions.

### CODE REVIEW

File: zc_install/sql/install/mysql_zencart.sql [hunk 1]
```diff
@@ -2451,9 +2451,9 @@ INSERT INTO configuration (configuration_title, configuration_key, configuration
...
-INSERT INTO configuration (configuration_title, configuration_key, configuration_value, val_function, configuration_description, configuration_group_id, sort_order, date_added) VALUES ('Upcoming Products ', 'MAX_DISPLAY_UPCOMING_PRODUCTS', '10', '{"error":"TEXT_MAX_ADMIN_DISPLAY_UPCOMING_PRODUCTS_LENGTH","id":"FILTER_VALIDATE_INT","options":{"options":{"min_range":0}}}', 'Number of \'upcoming\' products to display', '3', '6', now());
+INSERT INTO configuration (configuration_title, configuration_key, configuration_value, val_function, configuration_description, configuration_group_id, sort_order, date_added) VALUES ('Products on Special', 'MAX_DISPLAY_SPECIAL_PRODUCTS', '9', '{"error":"TEXT_MAX_ADMIN_DISPLAY_SPECIAL_PRODUCTS_LENGTH","id":"FILTER_VALIDATE_INT","options":{"options":{"min_range":0}}}', 'Number of products to display per page on the Specials page', '3', '4', now());
```
Comment (drbyte) [inline zc_install/sql/install/mysql_zencart.sql:2454]: The title here is missing the word "Page", whereas the upgrade SQL has the word "Page" in it.

File: zc_install/sql/install/mysql_zencart.sql [hunk 2]
```diff
@@ -2487,14 +2487,14 @@ INSERT INTO configuration (configuration_title, configuration_key, configuration
...
-INSERT INTO configuration (configuration_title, configuration_key, configuration_value, val_function, configuration_description, configuration_group_id, sort_order, date_added) VALUES ('Maximum Display of Featured Products Page', 'MAX_DISPLAY_PRODUCTS_FEATURED_PRODUCTS', '10', '{"error":"TEXT_MAX_ADMIN_DISPLAY_SEARCH_RESULTS_FEATURED_PRODUCTS_LENGTH","id":"FILTER_VALIDATE_INT","options":{"options":{"min_range":0}}}', 'Number of products to show per page when viewing Featured Products', 3, 29, now());
+INSERT INTO configuration (configuration_title, configuration_key, configuration_value, val_function, configuration_description, configuration_group_id, sort_order, date_added) VALUES ('Featured Products Centerbox', 'MAX_DISPLAY_SEARCH_RESULTS_FEATURED', '9', '{"error":"TEXT_MAX_ADMIN_DISPLAY_SEARCH_RESULTS_FEATURED_LENGTH","id":"FILTER_VALIDATE_INT","options":{"options":{"min_range":0}}}', 'Number of products to dispaly in the Featured Products centerbox', 3, 28, now());
```
Comment (drbyte) [inline zc_install/sql/install/mysql_zencart.sql:2490]: `dispaly` misspelled

**Comment by drbyte** [comment]
I noticed those 2 errors. Pls check the rest.

File: zc_install/sql/install/mysql_zencart.sql (hunk 1 above)
Comment (drbyte) [reply zc_install/sql/install/mysql_zencart.sql:2454]: Still mismatched:
`VALUES ('Products on Special', 'MAX_DISPLAY_SPECIAL_PRODUCTS`
vs
`UPDATE configuration SET configuration_title = 'Products on Special Page'`

**Comment by scottcwilson** [comment]
Should be ok now.  I noticed a couple of other listing page counts that hadn't been updated so I added those in.

**Pull request merged**